websites/H/havesexwith.me/presence.ts

```diff
@@ -0,0 +1,29 @@
+const state = ["Being epic", "Just browsing", "Chilling", "You should check this out", "Now this, is epic"][Math.floor(Math.random() * 3)];
+const smolimage = ["You should cop an email", "Hello there!", "Smol doggo"][Math.floor(Math.random() * 3)];
```

Comment:
yea PreMiD isn't for you to advertise, you need to add informational statuses stating what you are doing/ where you are on the page.
Else your presence will be denied.